deploy/haproxy-ingress.yaml

```diff
@@ -61,6 +61,14 @@ rules:
   - create
   - patch
   - update
+- apiGroups:
```

Comment:
We need to do the same for `deploy/haproxy-ingress-daemonset.yaml`

Reply:
True, the permissions are now also there.